packages/react-tinacms-editor/src/plugins/Image/Popups/Edit.tsx

```diff
@@ -285,14 +285,26 @@ const SaveLink = styled.button`
   }
 `
 
```

Comment:
Extending another styled component was somehow creating circular dependency issue.
Today it broke build, I guess some latest build changes thus I refactored it.